src/Components/Nav/Nav.js

```diff
@@ -42,8 +45,12 @@ export default class Nav extends Component {
     };
 
     render() {
+        const isEditMode = this.context && this.context.mode && this.context.mode === 'EDIT_MODE';
+        const remoteRendered = this.context && this.context.page && this.context.page.remoteRendered;
```

Comment:
I'm using `isEditModeFromDotCMS` for this to make the var name more self explanatory for others devs.

Reply:
?